changelog/5258.improvement.rst

```diff
@@ -0,0 +1,15 @@
+Added ``PikaEventBroker`` support for publishing to multiple queues. The former
+``queue`` key is deprecated. Queues should now be specified as a list in the
+``endpoints.yml`` event broker config under a new key ``queues``. Example config:
+
+.. code-block:: yaml
+
+  event_broker:
+    type: pika
+    url: localhost
+    username: username
+    password: password
+    queues:
+      - queue-1
+      - queue-2
+      - queue-3
```

Comment:
I'm not sure but i believe you're going to have to indent this more (4 spaces)

Reply:
it just has to be "one or more spaces", but most recommendations use two

Reply:
I just meant to make it render on the docs correctly